tools/proofers/character_selector.js

```diff
@@ -38,6 +38,16 @@ $(function () {
         mru = [];
     }
 
+    var pickerChars = {};
+    $('.picker').each(function pickerEach() {
+        pickerChars[$(this).text()] = true;
+    });
+
+    // filter out invalid characters
+    mru = mru.filter(function mruFilter(mruCharacter) {
+        return pickerChars[mruCharacter.character] === true;
+    });
+
```

Comment:
The functions could be anonymous.

Reply:
Sure, I am used to naming them for better stack traces, but this code isn't too complex.